spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -109,7 +109,7 @@ dependency for local SPI.
 
 ==== Local
 
-1. Open the spring-cloud-data-flow-admin-local/pom.xml in your IDE.
+1. Open the spring-cloud-data-flow-server-local/pom.xml in your IDE.
```

Comment:
looks like we have an extra hyphen here (between data and flow)... was already there, but just noticing it now